align_system/prompt_engineering/outlines_prompts.py

```diff
@@ -607,3 +607,33 @@ def comparative_kdma_score_prediction_json_schema(choices):
         "required": list(choices)
     }
     return json.dumps(json_schema)
+
+
+@outlines.prompt
+def scenario_description_classic_regression(scenario_state, character):
```

Comment:
I named this poorly, can you rename this something that makes it clear it's for your ADM?